src/Uno.UI/UI/Xaml/Style/Style.cs

```diff
@@ -91,15 +91,17 @@ internal void ApplyTo(DependencyObject o, DependencyPropertyValuePrecedences pre
 					}
 				}
 
+				localPrecedenceDisposable?.Dispose();
```

Comment:
Are we disposing twice now?

Reply:
We can do the refactor for 5x after https://github.com/unoplatform/uno/issues/11977